### DISCUSSION TITLE
Chat-ID für Telegram in welcher Zeile des Skripts

### DISCUSSION THREAD

**Discussion by electric-m** [camueller/SmartApplianceEnabler#180] [Installation und Betrieb]
So, ich sorge mal wieder dafür, dass es hier gefüllt wird.
Vielleicht interessiert es ja andere auch:

Wo genau im Shell Script muss die Telegram Chat ID eingetragen werden?
Über die Suche finde ich 4x das Wort Chat, verstehe aber nicht so recht, welche Stelle die richtige ist...

**Comment by camueller** [comment]
Blick in die Doku hilft: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Notifications_DE.md

**Comment by electric-m** [comment]
Nichts für ungut, aber die Doku habe ich  gelesen und auch soweit verstanden.
Darin steht jedoch nur, DASS die CHat ID eingesetzt werden soll, und nicht WO.

Die Elemente innerhalb des Skripts kann ich nicht deuten, bzw. wo dort genau die ID eingesetzt werden soll.

**Comment by camueller** [comment]
Im Script finden sich folgende Zeilen:
```
# vom @getidsbot ermittelte ID des Chats (bzw. der Gruppe) in den die Benachrichtigungen gepostet werden sollen
chat=
```
Eigentlich sollte klar sein, dass dort die ID einzutragen ist.

**Comment by electric-m** [comment]
Danke. Das fehlene "-ID" nach dem chat hat mich irritiert, aber klar, über die genannte Zähle hätte ich es vermuten können.
